docs/options.md

```diff
@@ -60,7 +60,7 @@ Use single quotes instead of double quotes.
 
 Notes:
 
-- Quotes in JSX will always be double and ignore this setting.
+- JSX Quotes will ignore this setting. see `jsx-single-quote`
```

Comment:
```suggestion
- JSX quotes ignore this option – see [jsx-single-quote].
```

Reply:
will update

Reply:
I think this new suggestions thing would have let you just press a button here on GitHub to apply the change..

Reply:
i've never used it before, but it seemed to want to do a commit for every applied suggestion when I tried, which seemed a bit excessive

Reply:
We always use “squash and merge” anyway so it doesn’t matter